### DISCUSSION TITLE
Something went wrong with my status bar

### DISCUSSION THREAD

**Discussion by ghost** [nvim-lualine/lualine.nvim#786] [Q&A] [answered]
**Environment **

- Nvim: 0.7.2
- OS : windows 11
- What is your airline configuration:  using [evil_lualine](https://github.com/nvim-lualine/lualine.nvim/blob/master/examples/evil_lualine.lua)

I tried to custom my status bar but i don't know why it went wrong with the icons and the shape of the sections holds its component. 
**_My status bar:_**
![e1](https://user-images.githubusercontent.com/95183956/183260852-3b47473d-a6a5-477c-b5a5-35cf57ff2301.png)
**_It's supposed to be like this:_**
**![e2](https://user-images.githubusercontent.com/95183956/183260957-8b38e2de-caac-47c2-94b4-c40c8bc4da0b.png)
_**Another example from vim-airline**_
![vin](https://user-images.githubusercontent.com/95183956/183261761-bca7896d-5be7-4b36-9a6f-696fb6119ac0.png)

**Comment by fitrh** [comment, marked as answer]
As the README state

> You'll also need to have a patched font if you want icons.

So you may need to install a nerd font and set it as your terminal emulator font
